ui/app/templates/application.hbs

```diff
@@ -18,6 +18,12 @@
           Pipelines
         </LinkTo>
       </div>
+      <div class="flex-auto"></div>
+      <div class="px-3 pb-4">
+        <a class="rounded bg-white inline-block" href={{concat this.apiURL "/openapi"}}>
```

Comment:
If you like, you can add a `target` attribute so that users will be able to still have Conduit and the swagger UI open in separate tabs when click the link:

```suggestion
        <a class="rounded bg-white inline-block" href={{concat this.apiURL "/openapi"}} target="_blank">
```

Reply:
done!